### PR TITLE
Fix incorrect 'driver_name' attribute values for EV3 UART sensors

### PR DESCRIPTION
**Pull request by in4lio**
Also `EV3_UART_SENSOR_NAME(suffix)` macros has been added into `ev3_uart_sensor.h`.

### CODE REVIEW

File: sensors/ev3_uart_sensor.h [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 #include <lego_sensor_class.h>
 
+#define EV3_UART_SENSOR_NAME(suffix) "lego-ev3-uart-" suffix
```
Comment (dlech) [inline sensors/ev3_uart_sensor.h]: The suffix is actually the "type ID" of the sensor, so can we replace `suffix` with `type_id`?

**Comment by dlech** [comment]
This looks great. Just the one nit-pick (about suffix/type_id). Also if you could squash the commits and force push, that would be better than having 2 or 3 commits for one change.

**Comment by in4lio** [comment]
Done.

**Comment by dlech** [comment]
Wonderful! Thank you very much!

**Pull request merged**